spec/cspec/c/kernel.mk

```diff
@@ -42,7 +52,7 @@ ifndef TOOLPREFIX
     endif
   endif
   ifdef TRY_TOOLPREFIX
-    TOOLPREFIX := $(firstword $(strip $(foreach TRY,${TRY_TOOLPREFIX},$(if $(shell which ${TRY}gcc),${TRY},))))
+    TOOLPREFIX := $(firstword $(strip $(foreach TRY,${TRY_TOOLPREFIX},$(if $(shell command -v ${TRY}gcc),${TRY},))))
```

Comment:
Happy to change it, but can you elaborate on what went wrong with `which`?

Reply:
On most platforms, `which` writes messages to stderr when it doesn't find anything, and not every platform has a `-s` silent option. In this part of the makefile, `which` was used in a way where it's likely to not find several things. The messages were a bit distracting when I was trying to figure out some other build problem, since it took me a couple of minutes to realise most of the messages were harmless.

Reply:
Cool, one more thing I learned today :-). Definitely happy with that change.